### PR TITLE
ER-1486 - Transfer without LegalEntityId +smever: patch

### PR DESCRIPTION
**Pull request by dev4valtech**
From original PR https://github.com/SkillsFundingAgency/das-recruit/pull/815

### CODE REVIEW

File: src/Jobs/Recruit.Vacancies.Jobs/ExternalSystemEventHandlers/UpdatedPermissionsExternalSystemEventsHandler.cs [hunk 1]
```diff
@@ -25,21 +26,20 @@ public class UpdatedPermissionsExternalSystemEventsHandler : IHandleMessages<Upd
...
-        private readonly IVacancyQuery _vacancyQuery;
         private readonly IMessaging _messaging;
+        
         private string ExternalSystemEventHandlerName => GetType().Name;
 
         public UpdatedPermissionsExternalSystemEventsHandler(ILogger<UpdatedPermissionsExternalSystemEventsHandler> logger, RecruitWebJobsSystemConfiguration jobsConfig,
                                                 IRecruitQueueService recruitQueueService,
                                                 IEmployerAccountProvider employerAccountProvider, IEncodingService encoder, IVacancyQuery vacancyQuery,
-                                                IMessaging messaging)
+                                                IMessaging messaging, IQueryStoreReader queryStoreReader)
```
Comment (shomavg) [inline src/Jobs/Recruit.Vacancies.Jobs/ExternalSystemEventHandlers/UpdatedPermissionsExternalSystemEventsHandler.cs]: vacancyQuery and queryStoreReader are not being used anywhere in the file.

File: src/Jobs/Recruit.Vacancies.Jobs/Jobs/TransferVacanciesFromProviderJob.cs [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
 using System.Linq;
 using System.Threading.Tasks;
 using Esfa.Recruit.Vacancies.Client.Application.Queues;
 using Esfa.Recruit.Vacancies.Client.Application.Queues.Messages;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore;
 
 namespace Esfa.Recruit.Vacancies.Jobs
 {
```
Comment (shomavg) [inline src/Jobs/Recruit.Vacancies.Jobs/Jobs/TransferVacanciesFromProviderJob.cs]: namespace can be corrected to point to the right collection.

Comment (dev4valtech) [reply src/Jobs/Recruit.Vacancies.Jobs/Jobs/TransferVacanciesFromProviderJob.cs]: Do you mean change to `namespace Esfa.Recruit.Vacancies.Jobs.Jobs`?

Comment (shomavg) [reply src/Jobs/Recruit.Vacancies.Jobs/Jobs/TransferVacanciesFromProviderJob.cs]: yes

**Pull request merged**